BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankNetworkingScreenApiCoordinator.swift

```diff
@@ -20,24 +20,40 @@ open class GiniBankNetworkingScreenApiCoordinator: GiniScreenAPICoordinator, Gin
     // MARK: - GiniCaptureDelegate
 
     public func didCapture(document: GiniCaptureDocument, networkDelegate: GiniCaptureNetworkDelegate) {
-        // The EPS QR codes are a special case, since they don0t have to be analyzed by the Gini Bank API and therefore,
-        // they are ready to be delivered after capturing them.
-        if let qrCodeDocument = document as? GiniQRCodeDocument,
-           let format = qrCodeDocument.qrCodeFormat,
-           case .eps4mobile = format {
-            let extractions = qrCodeDocument.extractedParameters.compactMap {
+        // Common logic for creating extractions
+        func createExtractions(for key: String, from document: GiniQRCodeDocument) -> [Extraction] {
```

Comment:
[Clarification]: Hi @igor-gini we never used the nested so far, currently we hide everything which not public behind the private functions what the benefits here?

Reply:
Linter was arguing for a long function, if needed I can move it to private

Reply:
That's a nice approach to silence the swiftlet warnings! 👏

Reply:
Awesome!